README.md

```diff
@@ -1,4 +1,6 @@
-[![Coverage Status](https://coveralls.io/repos/github/andela/ah-backend-thanos/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-backend-thanos?branch=develop)
+
```

Comment:
Why are you removing `coveralls`? Are you going to use `codeclimate`?

Reply:
You had already merged the pr that adds the coveralls file. so develop already has the coveralls file. this pr only adds the travis badge.